app/models/propertyrepresentation/ManageAgentOptions.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.propertyrepresentation
+
+import models.{EnumFormat, NamedEnum, NamedEnumSupport}
+import play.api.libs.json.{Json, OFormat}
+
+sealed trait ManageAgentOptions extends NamedEnum {
```

Comment:
what is the difference between these options and the above?

Reply:
answered above

Reply:
Shouldn't we be using Enumeration?